packages/test/test-end-to-end-tests/src/test/loadModes.spec.ts

```diff
@@ -273,15 +273,16 @@ describeCompat("LoadModes", "NoCompat", (getTestObjectProvider, apis: CompatApis
 			dataObject1.increment();
 		}
 		await loaderContainerTracker.ensureSynchronized(container1);
```

Comment:
Unrelated to your change. But these tests should probably be replaced to use provider APIs for creating container, loader or calling ensureSynchronized.